docs/UserGuide.md

```diff
@@ -88,9 +81,9 @@ Example:
 * `editPatient 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st patient to be `91234567` and `johndoe@example.com` respectively.
 * `editPatient 2 n/Betsy Crower t/` Edits the name of the 2nd patient to be `Betsy Crower` and clears all existing tags.
 
-### Listing all persons : `list`
+### Listing all patients: `list`
```

Comment:
I think for this part `deletePatient` should be before `list` and `find` since that is the order in the command summary table.
But maybe it would be better to reorder the command summary table instead since `deletePatient` requires you to use the `list` command first.

and also should change the `delete 2` and the `delete 1` in the examples to `deletePatient 2` and etc.